htdocs/web_portal/views/user/view_user.php

```diff
@@ -72,28 +72,38 @@
             </ul>
         </div>
 
-
+        <?php
+        $isUserAdmin = ($params['user'] && $params['user']->isAdmin());
+        ?>
         <!--  User -->
         <div class="tableContainer" style="width: 55%; float: left;">
             <span class="header" style="vertical-align:middle; float: left; padding-top: 0.9em; padding-left: 1em;">
                 User Details
             </span>
             <img src="<?php echo \GocContextPath::getPath()?>img/contact_card.png" class="decoration" />
             <table style="clear: both; width: 100%; table-layout: fixed;">
-                <tr class="site_table_row_1">
+            <?php if ($isUserAdmin) : ?>
+                <tr class="site_table_row_even">
+                    <td class="site_table">Last Logged In</td>
+                    <td class="site_table">
+                        <?php xecho($params['user']->getLastLoginDate()->format('Y-m-d H:i:s')) ?>
+                    </td>
+                </tr>
+            <?php endif; ?>
+                <tr class="site_table_row_<?php echo ($isUserAdmin) ? 'odd' : 'even' ?>">
```

Comment:
How does this line work? is it checking if isUserAdmin is true or false, making the row odd or even respectively?

Reply:
Right. It is checking true or false based on the even and odd will be appended.